packages/devtools_app/lib/src/shared/console/widgets/consolePane.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../common_widgets.dart';
+import '../../globals.dart';
+import '../../theme.dart';
+import '../console.dart';
+import '../console_service.dart';
+import 'evaluate.dart';
+
+// TODO(devoncarew): Show some small UI indicator when we receive stdout/stderr.
+
+@override
+PreferredSizeWidget buildConsolePaneHeader() {
```

Comment:
why does this top level method have an override? I would switch it to be a 
ConosolePaneHeader widget.